docs/natspec-format.rst

```diff
@@ -60,9 +60,34 @@ The following example shows a contract and a function using all available tags.
         /// @dev The Alexandr N. Tetearing algorithm could increase precision
         /// @param rings The number of rings from dendrochronological sample
         /// @return age in years, rounded up for partial years
-        function age(uint256 rings) external pure returns (uint256) {
+        function age(uint256 rings) external virtual pure returns (uint256) {
             return rings + 1;
         }
+
+        /// @notice Returns the amount of leaves the tree has.
+        /// @dev Returns only a fixed number.
+        function leaves() external virtual pure returns(uint256) {
+            return 2;
+        }
+    }
+
+    contract Plant {
+        function leaves() external virtual pure returns(uint256) {
+            return 3;
+        }
+    }
+
+    contract SubTree is Tree, Plant {
+        function age(uint256 rings) external override pure returns (uint256) {
+            return rings + 2;
+        }
+
+        /// For multiple inheritance we need to explicitly specify the base
```

Comment:
This sentence goes into the `@notice` tag, so it should describe what the function does.

By the way, do we have a test that checks that leaving out the `@notice` still works the same way with default and explicit inheritance?

Reply:
Yes we have such a test, see `dev_explicit_inherit`